apps/ledger-live-desktop/tests/specs/onboarding/connect-device.spec.ts

```diff
@@ -25,6 +25,7 @@ test.describe.parallel("Onboarding", () => {
       await test.step("Get started", async () => {
         await onboardingPage.getStarted();
         await onboardingPage.hoverDevice(Nano.nanoS);
+        await new Promise(r => setTimeout(r, 300)); // wait for the animation to finish
```

Comment:
not sure about this. maybe there is a `delay` helper if i'm not mistaking. Also, when running with playwright i think we disable the animations ? Maybe someone from the hub team can give us more infos,